### ISSUE TITLE
Completely wrong

### ISSUE THREAD

**Issue by AshleyAS** [MicrosoftDocs/visualstudio-docs#9752]
Nothing in this article matches Visual Studio's project creation past step 2. Starting at step 3...
Section 1, step 3: article lists the project names should be AngularWithASP and AngularWithASP.Server. Visual Studio creates anguarlapp and webapi

Section 2, "Set the project properties", step 2: there is no "Launch Browser" selection in either of the project properties.

Section 3: "Start the project" this does work

Section 4: "Publish the project": There is no ReferenceOutputAssembly value in the .csproj file and when I add it I get an error message stating "Error: NU1105	Unable to find project information for '[my directory]\source\repos\AngularWithASP\angularapp\angularapp.esproj'. If you are using Visual Studio, this may be because the project is unloaded or not part of the current solution so run a restore from the command-line. Otherwise, the project file may be invalid or missing targets required for restore.

At that point, I gave up using these instructions. 

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 4286e883-bc14-0f1e-1262-08e8432b1b46
* Version Independent ID: 09df456f-db19-ad41-9d7f-fa81ddb33c92
* Content: [Create an ASP.NET Core app with Angular - Visual Studio (Windows)](https://learn.microsoft.com/en-us/visualstudio/javascript/tutorial-asp-net-core-with-angular?view=vs-2022)
* Content Source: [docs/javascript/tutorial-asp-net-core-with-angular.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/main/docs/javascript/tutorial-asp-net-core-with-angular.md)
* Product: **visual-studio-windows**
* Technology: **vs-javascript**
* GitHub Login: @Mikejo5000
* Microsoft Alias: **mikejo**

**Comment by WilliamAntonRohm** [comment]
@AshleyAS -- Ashley, thank you for your feedback.  Please consider these resources:
- Ask a question or find an answer on the [Microsoft Q&A](https://learn.microsoft.com/en-us/answers/tags/176/vs) page for Visual Studio.
- Check the [Visual Studio MSDN Forum](https://social.msdn.microsoft.com/Forums/vstudio). 
- [Stack Overflow](https://stackoverflow.com/search?q=%22visual+studio%22)  
- [Reddit](https://www.reddit.com/r/VisualStudio)

@Mikejo5000 -- Mike, please look into this issue.

**Comment by Mikejo5000** [comment]
@AshleyAS Hi, thanks for the feedback. What version of Visual Studio are you using? We may need to clarify the version in the article, as the templates are getting updated pretty frequently right now.

**Comment by Mikejo5000** [comment]
I'm updating the article requirements. 17.8 Preview 2 or later is required for a few of the particular steps.

**Comment by AshleyAS** [comment]
I'm using Microsoft Visual Studio Professional 2022 (64-bit), Version 17.7.2

**Comment by Mikejo5000** [comment]
17.8 will be available as an upgrade option when 17.8 goes to GA. If you prefer, you can install Preview builds separately using this link: https://visualstudio.microsoft.com/vs/preview/. 
However, you already identified the differences w/the 17.7 template. (1) In 17.7, the two projects are named (by default) anguarlapp and webapi. And (2) the project reference to the angular project has to be manually added instead of the **Add > Project Reference**. The reference should go in the ItemGroup section of the .csproj file, but please verify the path used.
```
  <ItemGroup>
    <ProjectReference Include="..\angularapp\angularapp.esproj">
      <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
    </ProjectReference>
  </ItemGroup>
```
There is a **Launch Browser** option in all versions of VS 2022. From the webapi project properties, you need to first select **Open debug launch profiles UI** in the Debug tab before you can see **Launch Browser**.

**Comment by Mikejo5000** [comment]
Requirements updates related to 17.8 Preview 2 will be live by EOD PST. 
MicrosoftDocs/visualstudio-docs-pr#11831
